qasources.md

```diff
@@ -0,0 +1,50 @@
+## Main Endpoint
+**GET /api/integration/qasources**
+
+Provide access to the Quality Assurance sources. It returns the list of the Quality Assurance sources.
+
+```json
+[
+
+  {
+    id: "openaire",
+    type: "qasource",
+    totalEvents: "33"
+  },
+  {
+    id: "another-source",
+    type: "qasource",
+    lastEvent: "2020/10/09 10:11 UTC",
+    totalEvents: "21"
+  },
+  ...
+]
+```
+Attributes:
+* lastEvent: the date of the last update from the specific Quality Assurance source
+* totalEvents: the total number of suggestions provided by the Quality Assurance source
+* id: is the identifier to use in GET Single Source
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions, only system administrators can access
+
+## GET Single Source
+**GET /api/integration/qasources/<:qasource-id>**
```

Comment:
Hi @LucaGiamminonni,

One thing I recall from our meeting was that we would create a generic mechanism for quality assurance. 
Does it makes sense to have something like we have in authority or external sources for dealing with multiple sources?
https://github.com/DSpace/RestContract/blob/main/external-authority-sources.md#single-entry

`GET /api/integration/qasources/<:source-name>/entryValues/<:entry-id>`

in this specific case, the `notification broker (nb)` source-name?

Reply:
Thank you @LucaGiamminonni for your comment. Perhaps we are in different mindset regarding this. I look at this as new generic feature that we will be available in DSpace that handles sources, events, topics for quality assurance, being source agnostic. Apart from the implementation your team already have, I see it to be very similar to external sources and authority and how it should be specified. Perhaps external entities and authority should be differently and could be based also on search/bySource. But for the matter of the scope of this PR, for coherency and consistency, to me, this specification should be very close to what we already have regarding handling different sources.